assets/src/edit-story/app/layout/useZoomSetting.js

```diff
@@ -75,27 +73,6 @@ function calculateViewportProperties(
   workspaceSize,
```

Comment:
^ There seems to be still `hasCanvasZoom` in use, e.g. on the above line and also in the `LayoutProvider` when looking at the code.

Should we remove it fully since the feature flag was removed now?

Reply:
Good catch, fixed in 959ec53b4.